packages/tracing-enriched-data-handler/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "pagopa-interop-tracing-enriched-data-handler",
+  "version": "1.0.0",
+  "description": "PagoPA tracing enriched data handler",
+  "main": "dist",
+  "type": "module",
+  "scripts": {
+    "test": "vitest --config ./test/vitest.config.ts",
+    "test:it": "vitest db --config ./test/vitest.config.ts",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:autofix": "eslint . --ext .ts,.tsx --fix",
+    "format:check": "prettier --check src",
+    "format:write": "prettier --write src",
+    "start": "node  --no-warnings --loader ts-node/esm -r 'dotenv-flow/config' ./src/index.ts",
+    "build": "tsc"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^6.14.0",
```

Comment:
let's avoid using `^` or other modifiers. Instead we need to use the specific version.
It helps with renovate or other bots when they bump dependencies versions